lib/common/transformGroups.js

```diff
@@ -322,4 +322,14 @@ export default {
    * @memberof TransformGroups
    */
   'react-native': ['name/camel', 'color/css', 'size/object'],
+
+  /**
+   * Transforms:
+   *
+   * [attribute/cti](/reference/hooks/transforms/predefined#attributecti)
+   * [name/human](/reference/hooks/transforms/predefined#namehuman)
+   *
+   * @memberof TransformGroups
+   */
+  markdown: ['attribute/cti', 'name/human'],
```

Comment:
Same as my other comment, I don't think we need a transformGroup, first one seems redundant and the second one makes more sense to me if it matches the user's transformGroup's name transform (e.g. if they output to CSS they'll likely prefer kebab case for their markdown format so it somewhat matches with the tokens in code, they may even put a custom name transform kebab that also adds "--" prefix to be fully matching)